description/launch/simulate.launch

```diff
@@ -18,6 +17,11 @@
 
     </include>
 
+    <!-- Camera Plug-in -->
```

Comment:
the plugin is called twice: once from simulate.launch and once from spawn.launch.
spawn.launch is called in simulate.launch.